src/pynwb/data/nwb.image.yaml

```diff
@@ -81,10 +80,10 @@ groups:
     - num_files
     shape:
     - null
-    doc: 'Path or URL to one or more external file(s). Field only present if format=external.
-      NOTE: this is only relevant if the image is stored in the file system as one
+    doc: Paths or URLs to one or more external file(s). Field only present if format=external.
+      This is only relevant if the image is stored in the file system as one
       or more image file(s). This field should NOT be used if the image is stored
-      in another HDF5 file and that file is HDF5 linked to this file.'
+      in another NWB file and that file is HDF5 linked to this file.
```

Comment:
Maybe we shouldn't mention HDF5 in the schema, since this is supposed to back independent of backend